html/comparison.html.haml

```diff
@@ -115,7 +115,7 @@
                   Changed
                   = ": {{diff.packages.changed.length}} packages"
                 %span.summary-part{"ng-show" => "diff.packages.common"}
-                  %a{:class=>"show-common-elements", href: "#packages_both"}
+                  %a.both_link{:class=>"show-common-elements", href: "#packages_both"}
```

Comment:
why does the show-common-elements class needs to be inside the curly braces? couldn't we just use a.show-common-elements.both_link ?